packages/docusaurus-theme-search/src/theme/SearchBar/index.module.css

```
@@ -0,0 +1,703 @@
+.MarkpromptButton {
+  all: unset;
+  display: flex;
+  cursor: pointer;
+  border-radius: 99999px;
+  color: var(--markprompt-primaryForeground);
+  background-color: var(--markprompt-primary);
+  padding: 0.75rem;
+  position: fixed;
+  bottom: 2rem;
+  right: 2rem;
+  transition-property: opacity;
+  transition-duration: 200ms;
+}
+
+.MarkpromptButton:hover {
+  opacity: 0.8;
+}
+
+.MarkpromptIcon {
+  width: 1.25rem;
+  height: 1.25rem;
+}
+
+.MarkpromptOverlay {
+  position: fixed;
+  inset: 0;
+  animation: overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+  background-color: var(--markprompt-overlay);
+  z-index: calc(var(--ifm-z-index-fixed) + 1);
+}
+
+.MarkpromptContent {
+  background-color: var(--markprompt-background);
+  border-radius: var(--markprompt-radius);
+  border: 1px solid var(--markprompt-border);
+  box-shadow: hsl(206 22% 7% / 35%) 0px 10px 38px -10px,
+    hsl(206 22% 7% / 20%) 0px 10px 20px -15px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 80vw;
+  max-width: 600px;
+  height: calc(100vh - 200px);
+  max-height: 600px;
+  animation-name: contentShow;
+  animation-duration: 300ms;
+  animation-fill-mode: both;
+  transition-timing-function: cubic-bezier(0.25, 0.4, 0.55, 1.4);
+  color: var(--markprompt-foreground);
+  overflow: hidden;
+  display: grid;
+  grid-template-rows: auto 1fr;
+}
+
+.MarkpromptClose {
+  all: unset;
+  position: absolute;
+  top: 0.75rem;
+  right: 0.75rem;
+  color: var(--markprompt-foreground);
+  width: var(--markprompt-button-icon-size);
+  height: var(--markprompt-button-icon-size);
+  padding: 0.25rem;
+  border-radius: 4px;
```

```suggestion
  border-radius: 0.25rem;
```

We probably want the `border-radius` to scale in proportion when a user zooms in their browser.